Azure/connector_azure_network_watcher_collector.json

```
@@ -0,0 +1,48 @@
+{
+  "arguments": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "connector Azure Network Watcher configuration",
+    "type": "object",
+    "properties": {
+      "container_name": {
+        "description": "Container name to interact with Azure Network Watcher",
+        "type": "string"
+      },
+      "account_name": {
+        "description": "Account name of the Azure Network Watcher",
+        "type": "string"
+      },
+      "account_key": {
+        "description": "Account key of the Azure Network Watcher",
+        "type": "string"
+      },
+      "intake_server": {
+        "description": "Server of the intake server (e.g. 'https://intake.sekoia.io')",
+        "default": "https://intake.sekoia.io",
+        "type": "string"
+      },
+      "intake_key": {
+        "description": "Intake key to use when sending events",
+        "type": "string"
+      },
+      "chunk_size": {
+        "type": "integer",
+        "description": "The max size of chunks for the batch processing",
+        "default": 1000
+      }
+    },
+    "required": [
+      "container_name",
+      "account_name",
+      "account_key",
+      "intake_key"
+    ],
+    "secrets": [
+      "account_key"
+    ]
+  },
+  "description": "connector playbook to get Azure Network Watcher events information",
+  "docker_parameters": "azure_network_watcher",
+  "name": "[BETA] Collect Azure Network Watcher events",
+  "uuid": "3e5ff414-f54a-4c3e-8127-82177b31e3c4"
```

```suggestion
  "uuid": "626f4d84-5ab7-11ee-8c99-0242ac120002"
```
Azure network watcher should take the old uuid of Azure blob storage